### PR TITLE
Add automation details to the ingestion sources UI

### PR DESCRIPTION
**Pull request by axmb**
Couple notes:

- Considered doing something with "better" UX around schedule expression, but ended up reversing that. For lack of a bit more thought on the UX, we should probably punt on this for now.
- Not including AWS-esque validation on the schedule. There's a medium-sized regex in the Curator API data layer we can share here, but wasn't totally happy with just copy-pasta'ing that. Can revisit this later on.
- Excuse the inclusion of Docker changes, which I've added to locally E2E test.

### CODE REVIEW

File: verification/curator-service/ui/src/components/SourceTable.tsx [hunk 1]
```diff
@@ -59,6 +61,10 @@ interface TableRow {
     name: string;
     // origin
     url: string;
+    // automation
+    // schedule
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/SourceTable.tsx:65]: nit: should //schedule go below awsRuleArn?

Comment (axmb) [reply verification/curator-service/ui/src/components/SourceTable.tsx:65]: Both of these fields are in automation.schedule. Sorry if this is a strange way to denote that -- I'll change the comment format.

**Comment by axmb** [comment]
[Add source automation schedule data to the ingestion source schema/API/UI](https://trello.com/c/xzbc0sdK/176-add-source-automation-schedule-data-to-the-ingestion-source-schema-api-ui)

**Comment by axmb** [comment]
[Integrate read/update/delete methods of the ingestion source API with AWS](https://trello.com/c/iKw7u0U1/183-integrate-read-update-delete-methods-of-the-ingestion-source-api-with-aws)

**Pull request merged**